src/EnergyPlus/OutputProcessor.cc

```diff
@@ -1920,7 +1920,7 @@ namespace OutputProcessor {
                 OutResourceType = "Electricity";
 
             } else if (meterType == "GAS" || meterType == "NATURALGAS") {
-                OutResourceType = "Gas";
+                OutResourceType = "NaturalGas";
```

Comment:
For later - remove synonyms from the if statements in this function (gas, electric, propanegas, etc.)